docs/Guides/Replication.md

```diff
@@ -96,11 +96,11 @@ for component in replicatedComponents do
 			changes[key][name] = { data = record.new }
 		end
 	end
+end
 
-	-- Check if there are any changes in our buffer before sending the changes to all clients.
-	if next(changes) then
-		RemoteEvent:FireAllClients(changes)
-	end
+-- Check if there are any changes in our buffer before sending the changes to all clients.
+if next(changes) then
+	RemoteEvent:FireAllClients(changes)
```

Comment:
It looks like there's a formatting problem.

Reply:
Are you sure? It looks fine for me.

Reply:
It might be. Sorry, the super-wide tabs in GitHub are throwing me off